File: bzl/examples/android/MainActivity.java
package com.nianticlabs.bzl.examples.android;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import com.nianticlabs.bzl.examples.android.ExampleMethods;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        System.loadLibrary("hello-android");
        setContentView(R.layout.activity_main);
        TextView textView  = (TextView) findViewById(R.id.main_content);
        textView.setText(textView.getText() + "\n" + ExampleMethods.exampleString());
    }
}

File: bzl/examples/android-kotlin/main/java/com/example/kotlinactivityapp/MainActivity.kt
package com.example.kotlinactivityapp

import android.os.Bundle
import android.app.Activity
import android.os.Build

class MainActivity : Activity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.main)
    print(Build.VERSION_CODES.UPSIDE_DOWN_CAKE)
  }
}
